codegen-sbt/src/sbt-test/codegen/test-compile/test

```diff
@@ -1,14 +1,24 @@
-$ exists project/schema.graphql
-$ absent src/main/scala/Client.scala
 $ absent src/main/scala/client/ClientNameUniqueness.scala
+$ absent src/main/scala/client/ClientGitLab.scala
 $ mkdir src/main/scala
 $ mkdir src/main/scala/client
 
-> calibanGenClient project/schema.graphql src/main/scala/Client.scala
-$ exists src/main/scala/Client.scala
 > calibanGenClient project/schema-to-check-name-uniqueness.graphql src/main/scala/client/ClientNameUniqueness.scala --packageName client
 $ exists src/main/scala/client/ClientNameUniqueness.scala
 > calibanGenClient project/gitlab-schema.graphql src/main/scala/client/ClientGitLab.scala --packageName client
 $ exists src/main/scala/client/ClientGitLab.scala
 
+$ mkdir src/main/scala/genview
+$ mkdir src/main/scala/genview/client
+
+> calibanGenClient project/schema-to-check-name-uniqueness.graphql src/main/scala/genview/client/ClientNameUniqueness.scala --packageName genview.client --genView true
+$ exists src/main/scala/genview/client/ClientNameUniqueness.scala
+$ exec sh verify.sh StarshipView ./src/main/scala/genview/client/ClientNameUniqueness.scala
+> calibanGenClient project/gitlab-schema.graphql src/main/scala/genview/client/ClientGitLab.scala --packageName genview.client --genView true
```

Comment:
Let's keep only this gitlab one and remove the one without `--genView`, I think it's enough.